Module 4

kateblankenship1 opened the pull request:
Pull request.

kateblankenship1 committed:
added files

kateblankenship1 committed:
testing

testing

kateblankenship1 committed:
edited images

edited images

kateblankenship1 committed:
added stylesheet to other pages

kateblankenship1 committed:
trying to fix header img

kateblankenship1 committed:
trying to fix image

kateblankenship1 committed:
fixed images on dog page

jimmynono commented on blog.html:
```diff
@@ -0,0 +1,80 @@
+  <h2>Adoptadog Blog</h2>
+</div>
+
+<article class="blogarticles">
```
when using multi word css classes it's best to use kabob casing, where everything is lowercase and words are seperated by a dash.

```
<article class="super-exciting-fun-times">

jimmynono commented on blog.html:
```diff
@@ -0,0 +1,80 @@
+          long water shoob fat boi long water shoob much ruin diet, lotsa pats doggorino.
+        </p>
+  </section>
+  <section class="blogthumb"><img src="images/blog-2.jpg" alt="blog photo" />
```
Try to be more descriptive with your alt tags.

jimmynono commented on dogs.html:
```diff
@@ -0,0 +1,148 @@
+  <h2>Here are all of our dogs!</h2>
+</div>
+
+<article class="dogprofiles">
```
I'd argue that the wrapping container is a section, and the dog tiles are more articles than sections (or a div)

jimmynono commented on styles/stylesheet.css:
```diff
@@ -0,0 +1,135 @@
+z* {
```
I think you have an extra z here.

jimmynono commented on styles/stylesheet.css:
```diff
@@ -0,0 +1,135 @@
+}
+
+nav ul {
+  padding: 18px 0;
```
just for consistency, keep to rems for padding/margins to stay with your font sizes. Just a little cleaner.

jimmynono commented on styles/stylesheet.css:
```diff
@@ -0,0 +1,135 @@
+
+.dog {
+  display: block;
+  width: 22em;
```
These aren't exactly responsive. I'd lean you to using some sort of percent based system to keep them behaving properly. In your page I only see 2 dog tiles wide on a laptop, and we should see three. It also keeps them in ration with the rest of the page if you go with a percent based system.

kateblankenship1 replied:
Okay! I shall work on integrating that when I work on the next assignment!

jimmynono commented on styles/stylesheet.css:
```diff
@@ -0,0 +1,135 @@
+  width: 100%;
+}
+
+/*stuff here */
```
:)

kateblankenship1 committed:
changes made

Finished dog page.
Adjusted buttons to expand in the center of the dog profiles.
Added a div around these buttons for better customization.
Fixed navigation on all pages.
Added footer on all pages.
Switched all px to em or rem.

kateblankenship1 committed:
more changes

Fixed alt tags to be more descriptive.
Rearranged blog page in order to easily style it as needed.
Finished styling blog page.
Switched to kebab naming system, for those that apply.

kateblankenship1 committed:
checkout page

(almost) finished coding checkout page.

kateblankenship1 committed:
javascript - module 4

Created javascript file and linked to pages.
Altered dog photo classes to include the appropriate information for js.
Altered Adopt links to include the appropriate information for js.
Redid all the alt tags on the dog photos to be more descriptive.

kateblankenship1 committed:
module 6

added grid layout and made responsive.

kateblankenship1 committed:
module 7

fixed all images to be sized appropriately, rearranged a few items and changed some of my css properties in order to even out my layout.

Added gradient bg to header image.

Changed size of header image logo and headlining text so that it looks better on mobile.

kateblankenship1 committed:
lesson 8

Fixed my checkout page because my CSS somehow got screwed up for it and none of my text fields were showing. Actually ended up rewriting most of it to make it tidier.

Attempted to add javascript to make blog posts show up; successful with all except paragraphs, and for some reason mainHeader keeps repeating.

Couldn't get the getElementsByClassName to work so had to change to using #id, which did work for some reason.

kateblankenship1 committed:
lesson 8

submit button on form stopped working suddenly for some reason. Fixed. Was able to console log form values. Added general jquery code to the checkout page for this to work.

Still not able to get blog paragraphs to show up unfortunately!

kateblankenship1 committed:
milestone 3

Added jquery script to index and dogs.
Created dogs.js file for javascript.
Updated cart totals to equal 0 on all pages.

Updated dog click function to add the costs to the cart! I can't BELIEVE IT WORKED I feel like a wizard

Added Jquery effect - when you hover with the mouse over a dog's profile, the background turns a pale blue color.

kateblankenship1 committed:
module 10

Added testing.txt and the readme file.

kateblankenship1 committed:
updated testing.txt

Rewrote alot of my information on an afterthought to simulate a more professional document.

jimmynono commented on styles/stylesheet.css:
```diff
@@ -0,0 +1,446 @@
+
+/*BLOG PAGE */
+
+#blog-post-container {
```
it's generally not good practice to use id's for CSS. It's totally approprate to create a single class. We try to use ID's for javascript needs.

jimmynono commented on index.html:
```diff
@@ -0,0 +1,172 @@
+  </nav>
+<div class="format">
+  <div class="main-body">
+<div class="main-section">
```
Watch your indenting, it's a lot easier to see what's nested in elements if they are indented right.
